tests/test_global_opts.rs

```diff
@@ -157,9 +157,9 @@ fn test_resolve_workspace_directory() {
     // Ancestor of cwd
     let stdout = test_env.jj_cmd_success(&subdir, &["status"]);
     insta::assert_snapshot!(stdout, @r###"
-    Parent commit: zzzzzzzz 00000000 (empty) (no description set)
-    Working copy : qpvuntsm 230dd059 (empty) (no description set)
     The working copy is clean
+    Working copy : qpvuntsm 230dd059 (empty) (no description set)
+    Parent commit: zzzzzzzz 00000000 (empty) (no description set)
```

Comment:
I slightly prefer the original summary + diff order because I parse them as header + details.

```
Working copy :
Parent commit:
Working copy changes:
M ...
```

FWIW, maybe we have no tests for `jj status` in dirty working copy?

Reply:
I found the line about the parent commit sandwiched in between lines about the working copy confusing, especially when it's just:

```
Working copy :
Parent commit:
The working copy is clean.
```

My first instinct was just to remove "The working copy is clean." line and rely on the `(empty)`, but I think having an English sentence is friendly to beginners.

I do think

```
The working copy is clean.
Working copy :
Parent commit:
```

is nice.

At first, I also thought having the diff before the commit info isn't great, but I got used to it pretty quickly. OTOH, I haven't been a big user of `jj status` before (I might be becoming one now), so I don't know.

Another option is to always insert a newline:

```
Working copy :
Parent commit:

The working copy is clean.
```

I'm not sure whether I like that.

Reply:
`jj new` prints stats in that order (without empty line), but I don't feel it's odd.
```
Working copy now at: 4fb454f658fd (no description set)
Parent commit      : 48b1a1c533f1 working_copy: in ignored directories, visit only already tracked paths
Added 0 files, modified 2 files, removed 0 files
```

Anyway, I don't have strong opinion either. It's just a matter of taste.

Reply:
I still feel like it's friendlier for beginners to put the diff in the beginning for `status`. I might be wrong, but I'm currently thinking of merging this as is.

If it keeps bothering us, we can put the diff and conflicts after the working copy/parent commit again.

Reply:
Sure, we can try it and see